debug-panel-core/src/main/kotlin/com/redmadrobot/debug_panel_core/util/DebugPanelNotification.kt

```diff
@@ -51,7 +54,13 @@ internal class DebugPanelNotification(private val context: Context) {
             )
             .build()
 
-        notificationManager?.notify(NOTIFICATION_ID, notification)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            if (ContextCompat.checkSelfPermission(context, POST_NOTIFICATIONS) == PERMISSION_GRANTED) {
+                notificationManager?.notify(NOTIFICATION_ID, notification)
+            }
+        } else {
+            notificationManager?.notify(NOTIFICATION_ID, notification)
+        }
```

Comment:
I think it will be better to move this check to start of the function.

```kotlin
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU && 
    ContextCompat.checkSelfPermission(context, POST_NOTIFICATIONS) != PERMISSION_GRANTED) {
    return
}
```

We don't want to build permission if it is not going to be shown.